src/plugins/stacked-barchart/README.md

```diff
@@ -0,0 +1,44 @@
+# Grafana Panel Plugin Template
+
+[![Build](https://github.com/grafana/grafana-starter-panel/workflows/CI/badge.svg)](https://github.com/grafana/grafana-starter-panel/actions?query=workflow%3A%22CI%22)
```

Comment:
Oops. Copy/pasta fail :wink:

Reply:
Oh ok you didn't update the plugin template's readme yet.

Reply:
Indeed, I have forgotten about the readme, updated it now)